### ISSUE TITLE
Modularize the code

### ISSUE THREAD

**Issue by Samyak2** [varunp2k/WhatsappAutomation#12]
## Description

 - Code blocks like the logout code in #10 , send message, saving and loading LocalStorage etc are repeated in many of the scripts.
 - Changes made to one part in one of the scripts have to be copied to other scripts.

## Possible solution

 - Create a `utils.py` module that will contain functions for sending message, log out, load and save LocalStorage etc.
 - Import required functions in the scripts.
 - Edit: Create a separate directory for saving the modularized version of the code. Do not modify the original scripts.

**Comment by varunp2k** [comment]
I feel it's not necessary. Let people run the files individually. 
We can have another folder for having complete modularity.
What do you say?

**Comment by Samyak2** [comment]
Yes, it makes sense to have standalone scripts but modularity will help in organising the code well. Having both of these is a good idea, but it will have to be updated regularly.

**Comment by varunp2k** [comment]
Yes, I Agree with you.
Please edit the original description of the issue

**Comment by bhargavsk1077** [comment]
i'll work on this issue

**Comment by varunp2k** [comment]
Sure. Thank you.
Feel free to reach out if you are facing any issues
